### ISSUE TITLE
Expose more details when an `ERROR` happens

### ISSUE THREAD

**Issue by alessio-locatelli** [lycheeverse/lychee#1569]
## Version

```
lycheeverse/lychee:sha-18d486d-alpine@sha256:bb2a23d44ac713aaeea3def850dc6a2b2c57ba6dfba7f730c0088f670916345a
```

## Command example

```bash
lychee --exclude-path docs/broken.md --accept '200..=204,403,429,500,503' -vv 'docs/**/*.md' '**/*.yaml' '*.toml'
```

## Problem

The log says "Find details below"... but all the details is the `ERROR` word. It is difficult for a user to debug what has happened here. 

```shell
   [ERROR] https://www.privacyguides.org/en/news-aggregators/
Issues found in 1 input. Find details below.
[docs/tutorial.md]:
   [ERROR] https://www.privacyguides.org/en/news-aggregators/
🔍 357 Total (in 9s) ✅ 354 OK 🚫 1 Error

ERROR: Job failed: command terminated with exit code 2
```

## Expected behavior

Log the full traceback or a network error, not just the `ERROR` word.

**Comment by mre** [comment]
This should be fixed in `master` with https://github.com/lycheeverse/lychee/pull/1553. It will be released in 0.17.1 when this gets merged: https://github.com/lycheeverse/lychee/pull/1554.
In the meantime, can you try the latest `master` build and see if that resolves the issue?

**Comment by alessio-locatelli** [comment]
> This should be fixed in master with https://github.com/lycheeverse/lychee/pull/1553. It will be released in 0.17.1 when this gets merged: https://github.com/lycheeverse/lychee/pull/1554.

Sorry, I checked the issues, but not recently merged pull requests.

>  In the meantime, can you try the latest master build and see if that resolves the issue?

To be honest, I have no idea how to reproduce the error, as there were no details in the log, so I can only guess at what could go wrong. For example, the URL I show here was working, but `lychee` just said `ERROR` and I cannot know why.

Thank you for your reply.   I will re-open the issue if I get this problem again after `0.17.1`.

**Comment by mre** [comment]
Here's some sample output with the latest lychee on [privacyguides.org](https://github.com/privacyguides/privacyguides.org):

```md
[docs/email.md]:
   [ERROR] https://tuta.com/support#encrypted-address-book | Network error: error sending request for url (https://tuta.com/support#encrypted-address-book) Maybe a certificate error?
 [TIMEOUT] https://tuta.com/support/#pgp | Timeout
```

So the error is printed again, just like in the past. 😄
